org.jonata.Subtitld.yml

```diff
@@ -0,0 +1,503 @@
+              - "--enable-vp8"
+              - "--enable-postproc"
+              - "--enable-vp9"
+              - "--enable-vp9-highbitdepth"
+            sources:
+              - type: archive
+                url: https://github.com/webmproject/libvpx/archive/refs/tags/v1.11.0.tar.gz
+                sha256: 965e51c91ad9851e2337aebcc0f517440c637c506f3a03948062e3d5ea129a83
+          - name: x264
+            config-opts:
+              - "--prefix=${FLATPAK_DEST}"
+              - "--enable-shared"
+              - "--disable-cli"
+              - "--disable-lavf"
+              - "--enable-pic"
+            build-options:
+              arch:
+                aarch64:
+                  cxxflags: "-fPIC"
+                  config-opts:
+                    - "--disable-lto"
+                x86_64:
+                  config-opts:
+                    - "--disable-asm"
+            post-install:
+              - ln -srf ${FLATPAK_DEST}/lib/libx264.so.* ${FLATPAK_DEST}/lib/libx264.so
+            sources:
+              - type: git
+                commit: 1771b556ee45207f8711744ccbd5d42a3949b14c
+                url: https://git.videolan.org/git/x264.git
+          - name: x265
+            buildsystem: simple
+            build-options:
+              arch:
+                aarch64:
+                  cxxflags: "-fPIC"
+            build-commands:
+              - mkdir source/build-8 source/build-10 source/build-12
+              - cd source/build-12 && cmake .. -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} -DHIGH_BIT_DEPTH='TRUE' -DMAIN12='TRUE' -DEXPORT_C_API='FALSE' -DENABLE_CLI='FALSE' -DENABLE_SHARED='FALSE'
+              - cd source/build-12 && make
+              - cd source/build-10 && cmake .. -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} -DHIGH_BIT_DEPTH='TRUE' -DEXPORT_C_API='FALSE' -DENABLE_CLI='FALSE' -DENABLE_SHARED='FALSE'
+              - cd source/build-10 && make
+              - ln -s ../build-10/libx265.a source/build-8/libx265_main10.a
+              - ln -s ../build-12/libx265.a source/build-8/libx265_main12.a
+              - cd source/build-8 && cmake .. -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} -DENABLE_SHARED='TRUE' -DENABLE_CLI='FALSE' -DENABLE_HDR10_PLUS='TRUE' -DEXTRA_LIB='x265_main10.a;x265_main12.a' -DEXTRA_LINK_FLAGS='-L.' -DLINKED_10BIT='TRUE' -DLINKED_12BIT='TRUE'
+              - cd source/build-8 && make
+              - cd source/build-8 && make install
+            sources:
+            - type: archive
+              url: https://bitbucket.org/multicoreware/x265_git/downloads/x265_3.3.tar.gz
+              sha256: c6d744a87eda55560da715f56f878640554ddc06e2d0fcbd822fa330affc22cc
+          - name: fdkaac
+            config-opts:
+              - "--disable-static"
+            sources:
+              - type: archive
+                url: https://github.com/mstorsjo/fdk-aac/archive/v2.0.2.tar.gz
+                sha256: 7812b4f0cf66acda0d0fe4302545339517e702af7674dd04e5fe22a5ade16a90
+          - name: lame
```

Comment:
I do believe what's in the runtime ought to be enough

Reply:
Sorry, can you be more specific?

Reply:
lame is in the platform/sdk so you shouldn't need to build it.

Reply:
Oh, I see. And what about the hang on latest build, do you have any suggestion?